### PR TITLE
🦄 Menu revamp

### PR DESCRIPTION
**Pull request by bwnstck**
# Description
Fix design of Menu and rename
Fixes #34 

## Type of change

- [ x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: client/.eslintcache [hunk 1]
```diff
@@ -0,0 +1 @@
..._repos/DispoDisco/client/src/components/Header.js":"3","/Users/benji/dev/bwnstck_repos/DispoDisco/client/src/components/HeaderHome.js":"4"},{"size":765,"mtime":1606394314141,"results":"5","hashOfConfig":"6"},{"size":601,"mtime":1606392471295,"results":"7","hashOfConfig":"6"},{"size":1557,"mtime":1606394311825,"results":"8","hashOfConfig":"6"},{"size":695,"mtime":1606394306597,"results":"9","hashOfConfig":"6"},{"filePath":"10","messages":"11","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1hzgctc",{"filePath":"12","messages":"13","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"16","messages":"17","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"/Users/benji/dev/bwnstck_repos/DispoDisco/client/src/stories/Header.stories.js",[],"/Users/benji/dev/bwnstck_repos/DispoDisco/client/src/components/HeaderMain.js",[],"/Users/benji/dev/bwnstck_repos/DispoDisco/client/src/components/Header.js",[],"/Users/benji/dev/bwnstck_repos/DispoDisco/client/src/components/HeaderHome.js",[]]
```
Comment (phlgr) [inline client/.eslintcache]: Remove this from the git cache

File: client/.gitignore [hunk 2]
```diff
@@ -21,4 +21,3 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-client/.eslintcache
```
Comment (phlgr) [inline client/.gitignore]: You should only have a .gitignore in your root

File: client/src/components/HeaderHome.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+const Second = styled(First)`
+  grid-area: second;
+`;
+
+const Logo = styled.img`
+  margin-top: -50px;
+  filter: drop-shadow(0px 3px 6px var(--text-secondary));
+`;
+export const HeaderHome = () => (
```
Comment (phlgr) [inline client/src/components/HeaderHome.js:27]:
```suggestion
default export const HeaderHome = () => (
```

File: client/src/components/Header.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+`;
+const Title = styled.h2`
+  text-align: center;
+  font-size: clamp(1.7rem, 5vw, 5rem);
+`;
+const Children = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+export const Header = ({ title, children }) => (
```
Comment (phlgr) [inline client/src/components/Header.js:45]:
```suggestion
default export const Header = ({ title, children }) => (
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/.gitignore (hunk 2 above)
Comment (bwnstck) [reply client/.gitignore]: I already wondered where it comes from :)

File: client/src/components/Header.js (hunk 4 above)
Comment (bwnstck) [reply client/src/components/Header.js:45]: as reviewed by @domko95  we'll make a separate PR for this #68

File: client/src/components/HeaderHome.js (hunk 3 above)
Comment (macebm) [reply client/src/components/HeaderHome.js:27]: But please use **export default** **not** default export

File: client/src/components/Header.js (hunk 4 above)
Comment (macebm) [reply client/src/components/Header.js:45]: But use export default not default export

File: client/.eslintcache (hunk 1 above)
Comment (bwnstck) [reply client/.eslintcache]: i already tried this serveral times ... but somehow it comes back :) ill try it some more times with `git rm -f client/.eslintcache`

File: client/src/pages/MainMenu.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 import styled from "styled-components";
```
Comment (macebm) [inline client/src/pages/MainMenu.js:2]: styled-components/**macro**?

Comment (bwnstck) [reply client/src/pages/MainMenu.js:2]: yep, for better debugging 🤘🚀 its shows the original class-names while inspecting

**Review by arassaria — APPROVED** [review]
🦄 LGTM 🚀

**Pull request merged**